conf/messages

```diff
@@ -1459,6 +1459,16 @@ tai.updateIncome.CYPlus1.same.paragraph1=You have entered {0}
 tai.updateIncome.CYPlus1.same.paragraph2=This is the same as the amount already estimated for {0} for next year, so you do not need to update your income.
 tai.updateIncome.CYPlus1.same.cancel=Return to your Income Tax Summary
 
+tai.page_not_found.tabtitle = Page not found - Check your Income Tax - GOV.UK
```

Comment:
You don't need a separate tab title with "- Check your Income Tax - GOV.UK". You can just have tai.page_not_found.heading and that can be used for both title and h1. The main template already adds service title and GOV.UK at the end of the page title - If you look at the browser title now, it says "Page not found - Check your Income Tax - GOV.UK - Check your Income Tax - GOV.UK"

Reply:
cheers for the explanation regarding this, it had confused me when it printed Page not found - Check your Income Tax - GOV.UK - Check your Income Tax - GOV.UK", I had not released the 'Check your Income Tax - GOV.UK' would be appended to within the . main template for the heading field. In future I'll look out for this.